### PR TITLE
update nuxt with module usage

### PR DESCRIPTION
**Pull request by pi0**
Hi! 

Nitro and Nuxt maintainer here. I was contacted by @dario-piotrowicz and found out about this POC repo.

I wish knew earlier to be able to help but TLDR, both Nitro and Nuxt have a very powerful interface (Modules) for extending the core behavior and hooks API for context injections instead of middleware. 

Also soon we were already planning to directly support `devPreset: 'cloudflare'` (or `miniflare`) to natively use Minifilare when direct access to the platform natives is needed.

I made a module (https://github.com/pi0/nitro-cloudflare-bindings) and updated your POC use it. Also invited you to the repo and if you are available in Discord, would love to have you in our private channel for followup ❤️

### CODE REVIEW

File: apps/nuxt/nuxt.config.ts [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-        target: 'es2022'
-      },
-    },
-    rollupConfig: { 
-      external: ['wrangler']
-    }
-  },
-})
+  // https://github.com/pi0/nitro-cloudflare-bindings
+  modules: ["nitro-cloudflare-bindings"],
```
Comment (dario-piotrowicz) [inline apps/nuxt/nuxt.config.ts:5]: 😍

**Comment by jculvey** [comment]
This looks amazing! Nice to meet you and thanks for the update.

> Also soon we were already planning to directly support devPreset: 'cloudflare' (or miniflare) to natively use Minifilare when direct access to the platform natives is needed.

This sounds very exciting 😄.

**Comment by pi0** [comment]
(just released a new patch 0.0.6 🙈 )

**Comment by jculvey** [comment]
> (just released a new patch 0.0.6 🙈 )

lol woops! 

No prob I can bump it.

**Pull request merged**